plugins/woocommerce/src/Internal/Admin/Settings/PaymentsController.php

```diff
@@ -102,16 +114,9 @@ public function handle_feature_enablement_for_v9_8() {
 			return;
 		}
 
-		// If the feature is already enabled, we don't need to do anything.
-		if ( FeaturesUtil::feature_is_enabled( 'reactify-classic-payments-settings' ) ) {
-			return;
-		}
-
-		// If the feature is not enabled and the store is "empty" or did not enable any gateway yet,
+		// If the feature is not enabled and the store was created with 9.8.0 or 9.8.1,
 		// we will one-time force enable it.
-		if ( \WC_Install::is_new_install() || ! $this->store_has_enabled_gateways() ) {
-			wc_get_container()->get( FeaturesController::class )->change_feature_enable( 'reactify-classic-payments-settings', true );
-		}
+		wc_get_container()->get( FeaturesController::class )->change_feature_enable( 'reactify-classic-payments-settings', true );
```

Comment:
No need to discern "store existance". All 9.8.0 and 9.8.1 stores get the feature auto-enabled when they upgrade WC.